### PR TITLE
support ceph, auto-resume, auto_scale_lr and benchmark_test

### PR DESCRIPTION
**Pull request by hhaAndroid**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: .dev_scripts/benchmark_test.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+    config = Config.fromfile(args.config)
+
+    # test all model
+    logger = MMLogger.get_instance(
+        name='MMLogger',
+        log_file='benchmark_test.log',
+        log_level=logging.ERROR)
+
+    for model_key in config:
```
Comment (chhluo) [inline .dev_scripts/benchmark_test.py:141]: What does the config look like?

File: .dev_scripts/benchmark_train.py [hunk 2]
```diff
@@ -13,15 +13,69 @@
 from mmdet.utils import register_all_modules, replace_cfg_vals
 
 
+# TODO: Support full ceph
+def replace_to_ceph(cfg):
```
Comment (chhluo) [inline .dev_scripts/benchmark_train.py:17]: add a file to put this function, and import it in test and train scripts

Comment (hhaAndroid) [reply .dev_scripts/benchmark_train.py:17]: OK

File: .dev_scripts/benchmark_test.py (hunk 1 above)
Comment (hhaAndroid) [reply .dev_scripts/benchmark_test.py:141]: .dev_scripts/batch_test_list.py

File: .dev_scripts/benchmark_test.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        runner = Runner.from_cfg(cfg)
+    else:
+        # build customized runner from the registry
+        # if 'runner_type' is set in the cfg
+        runner = RUNNERS.build(cfg)
+
+    runner.test()
+
+
+# Sample test whether the train code is correct
```
Comment (chhluo) [inline .dev_scripts/benchmark_test.py:85]: train -> inference

**Review by ZwwWayne — DISMISSED** [review]

**Pull request merged**